### PR TITLE
Add Arc-Version Header

### PR DESCRIPTION
**Pull request by xelzi**
1. In README curl requests
2. In the server's arc client

I ran through the demo and one thing that surprised me -- `arc-version` did not come back on the request for an access token 🤔

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,24 +122,26 @@ The backend we started up is designed to print out data received at the [`/webho
...
+```.sh 
 HTTP/2 200
-date: Thu, 27 May 2021 00:57:07 GMT
+date: Fri, 29 Jul 2022 16:18:08 GMT
 content-type: application/json; charset=utf-8
-content-length: 190
+content-length: 288
 x-frame-options: ALLOWALL
-etag: W/"e525d19319fb4157b672cbbcfcd96630"
+arc-version: 2021-11-17
```
Comment (xelzi) [inline README.md:136]: 🎉

File: README.md [hunk 2]
```diff
@@ -163,20 +166,22 @@ npm start
...
 Your response should look something like this:
 ```.sh
 HTTP/2 202
-date: Thu, 27 May 2021 01:08:48 GMT
-content-type: text/html
+date: Fri, 29 Jul 2022 16:23:30 GMT
+content-type: application/json
 content-length: 0
 x-frame-options: ALLOWALL
+arc-version: 2021-11-17
```
Comment (xelzi) [inline README.md:180]: 🎉

File: README.md [hunk 3]
```diff
@@ -86,25 +86,26 @@ source .env
...
 pragma: no-cache
-etag: W/"2936966eeb8a0c0896610ede6dc6f015"
-x-request-id: 973379cc-6183-470f-ab96-777fda13c9bd
-x-runtime: 0.360905
-x-amzn-trace-id: Root=1-60aec757-21d4794f6c91c82269602213
+etag: W/"8c7f6bc44d7745bd8c56abfdec390be0"
+x-request-id: d7fa17eb-b6f1-421e-a83b-6a068973da47
+x-runtime: 0.230986
+x-amzn-trace-id: Root=1-62e40811-433e26ca14ef07465a4006b7
 vary: Origin
```
Comment (xelzi) [inline README.md:106]: this response missing the `arc-version` 🤔

Comment (joseph-dilallo) [reply README.md:106]: It looks like this controller doesn't inherit from `BaseController`:
```
module API::Enterprise::Auth
  class AccessTokensController < Doorkeeper::TokensController
```

Didn't realize that. We can fix this!

File: README.md [hunk 4]
```diff
@@ -121,24 +122,26 @@ The backend we started up is designed to print out data received at the [`/webho
...
-x-request-id: f68888b9-e34d-486e-996f-a51770a77fda
-x-runtime: 0.025782
-x-amzn-trace-id: Root=1-60aeee63-225b188311ad903f1c41b343
+x-request-id: f67bb652-a15a-4988-8e96-6d1c0d501a40
+x-runtime: 0.194169
+x-amzn-trace-id: Root=1-62e40840-70b526ce4db891c574a8b9d0
 vary: Origin
 
-{"id":"YOUR_ARC_WEBHOOK_ENDPOINT_ID","url":"ARC_TUNNELING_URL/webhook_listener","signing_key":"ARC_WEBHOOK_SIGNING_KEY","created_at":"2021-05-26T20:57:07.239-04:00","updated_at":"2021-05-26T20:57:07.239-04:00"}
+{"id":"ARC_WEBHOOK_ENDPOINT_ID","url":"ARC_TUNNELING_URL/webhook_listener"/webhook_listener","created_at":"2022-07-29T12:18:08.940-04:00","updated_at":"2022-07-29T12:18:08.940-04:00","sandboxed":true,"enabled":true,"arc_version":"2021-11-17","signing_key":"FCM1HxVsKHpAwIrC"}
```
Comment (joseph-dilallo) [inline README.md:144]: The signing key used to be blanked out with `ARC_WEBHOOK_SIGNING_KEY`. I do not know if that is important to maintain.

Comment (xelzi) [reply README.md:144]: yes thank you 💯

File: README.md (hunk 3 above)
Comment (xelzi) [reply README.md:106]: Cool! I can make the update :)

File: README.md [hunk 5]
```diff
@@ -86,25 +86,26 @@ source .env
 ```.sh
 curl -i -X POST https://api.arcadia.com/auth/access_token \
   -F "client_id=$ARC_API_CLIENT_ID" \
-  -F "client_secret=$ARC_API_CLIENT_SECRET"
+  -F "client_secret=$ARC_API_CLIENT_SECRET" \
+  -H "'Arc-Version': '2021-11-17'"
```
Comment (clifff) [inline README.md:90]: Probably only a concern for those of us that end up hand-typing these commands, but tested using `-H "Arc-Version: 2021-11-17` locally and seems to work without the single quotes

**Review by clifff — APPROVED** [review]
LGTM! If we want to hold until `Arc-Version` actually appears in the `/auth/access_token` response that seems fine but could write it into the documented response too so we aren't blocked on merging

**Pull request merged**